packages/ui/src/views/repo/pull-request/diff-viewer/pull-request-diff-viewer.tsx

```diff
@@ -294,7 +294,7 @@ const PullRequestDiffViewer = ({
                                 {getInitials(currentUser ?? '', 2)}
                               </AvatarFallback>
                             </Avatar>
-                            <Text color="primary">adam </Text>
+                            <Text color="primary">{currentUser} </Text>
```

Comment:
Since currentUser is nullable, I think we should have a null check.

Reply:
done